packages/filters/filter-color-matrix/src/ColorMatrixFilter.ts

```diff
@@ -141,6 +141,30 @@ export class ColorMatrixFilter extends Filter
         this._loadMatrix(matrix, multiply);
     }
 
+    /**
+     * Sets each channel on the diagonal of the color matrix.
+     * This achieves a tinting effect.
+     *
+     * @param {number} color - Color of the tint. This is a hex value.
+     * @param {boolean} multiply - if true, current matrix and matrix are multiplied. If false,
+     *  just set the current matrix with @param matrix
+     */
+    public tint(color: number, multiply: boolean): void
```

Comment:
Multiply could probably be optional since `_loadMatrix` sets a default of `false`.
```suggestion
    public tint(color: number, multiply?: boolean): void
```

Reply:
I followed other methods like `brightness`. But this can be done and makes more sense in false anyway

Reply:
I see that `saturate` its optional but most everything isn't. I think this should probably be optional for param for all that allow it. So I'd say make this change and we'll follow up with a fix for the typings here.